plugins/source/aws/docs/tables/aws_resiliencehub_component_recommendations.md

```diff
@@ -18,6 +18,8 @@ This table depends on [aws_resiliencehub_app_assessments](aws_resiliencehub_app_
 |_cq_parent_id|UUID|
 |account_id (PK)|String|
 |region (PK)|String|
+|app_arn (PK)|String|
+|assessment_arn (PK)|String|
```

Comment:
If we are adding `app_arn` doesn't that mean that `account_id` and `region` are duplicated?